app/models/warehouse/case_report.rb

```diff
@@ -125,7 +125,6 @@ def generate(kase)
         case_report.number = kase.number
         case_report.case_type = kase.decorate.pretty_type
         case_report.current_state = kase.decorate.status
-        case_report.rejected = kase.rejected? ? "Yes" : "No"
```

Comment:
Removed this with the thought we can rely on the case_originally_rejected flag and the current state for filtering rejected cases

Reply:
This attribute is being used in reports in the app so it's needed for that. (which is the reason it's been added)

The easiest way to filter out rejected cases will be to use this attribute, so whilst there might be another way to do it, it would be more complicated and require implementation logic and I'm not sure what the benefit would be.

In theory, regular users should be able to make any report they want in metabase, so having a boolean value for this is also easiest for them.